server.go

```diff
@@ -10,8 +10,7 @@ import (
 	"sync"
 	"time"
 
-	recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
-	tags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/recovery"
```

Comment:
Could you please sort the imports section? They should be grouped into: built-ins, external, local. This new line should stay within the external group